cmd/plugins/juju-wait-for/application.go

```diff
@@ -71,6 +74,7 @@ func (c *applicationCommand) SetFlags(f *gnuflag.FlagSet) {
 	c.waitForCommandBase.SetFlags(f)
 	f.StringVar(&c.query, "query", `life=="alive" && status=="active"`, "query the goal state")
 	f.DurationVar(&c.timeout, "timeout", time.Minute*10, "how long to wait, before timing out")
+	f.BoolVar(&c.summary, "summary", true, "output a summary of the application query if successful")
```

Comment:
Why not, if requested, print the summary on failure (time-out)? Then the issuer can see what it is at that point.

Reply:
Yeah, ok, that makes sense :)